corehq/apps/app_manager/templates/app_manager/v2/partials/custom_icon.html

```diff
@@ -0,0 +1,31 @@
+{% load xforms_extras %}
+{% load i18n %}
+
+<div>
+    <div class="form-group container-fluid">
+        <label class="control-label col-sm-2">{% trans "Custom Icon" %}</label>
+        <div class="col-sm-10 commcare-feature" data-since-version="1.3">
+            <label>
+                <span>
+                    {% trans "Form" %}
+                </span>
+                <input class="form-control" name="custom_icon_form" id="custom_icon_form"
+                       value="{% if custom_icon.form %}{{ custom_icon.form }}{% endif %}">
+            </label>
+            <label>
+                <span>
+                    {% trans "Text Body" %}
+                </span>
+                <input class="form-control" name="custom_icon_text_body" id="custom_icon_text_body"
+                       value="{{ custom_icon.text|trans:langs }}">
+            </label>
+            <label>
+                <span>
+                    {% trans "Xpath Function" %}
+                </span>
+                <input class="form-control" name="custom_icon_xpath" id="custom_icon_xpath"
```

Comment:
Will try to incorporate this in the follow up PR where i add the ability to add multiple custom icons